docs/dbaas/DBaaS_tepmplate.md

```diff
@@ -1,3 +1,139 @@
+# DatabaseCluster Templates
+
+DatabaseCluster Template allows operators to customize Database Clusters based on their requirements, environments, or infrastructure.
+
+Examples
+
+- **Data clusters**: Different load patterns, such as simple reads, heavy writes, 50%/50% read/write, and the number of connections.
+
+- **Infrastructure** - Different parameters and tuning for the resulting cluster: such as network configuration (load balancing, exposure), storage classes/types, etc.
+
+- **Environment**: Combination of database cluster and infrastructure that will affect the database cluster configuration.
```

Comment:
This seems to duplicate what we had previously
> # Create a database cluster from a template
> Database clusters can be created from templates using PMM. Templates allow you to customize the creation of database clusters. You can adapt templates to tweak K8s-specific settings such as **liveness probes**, changing **config maps**, or tuning **database engines**.

Reply:
Done.